duneggd/SubDetector/TMS.py

```diff
@@ -236,38 +253,67 @@ def construct(self, geom):
 
 
         # Scintillator
-        # Individual scintillator bar
+        # Individual scintillator bar width * number of bars
+        #0.01010526316*114
+        #0.02618181818*44
+        #0.03031578947*38
+        #0.04430769231*26
+        #0.0576*20
+        #0.064*18
+        #0.072*16
+        #0.288*4
+        barwidth=0.036
+        barnumber=32
+        middlepos=barwidth*barnumber/2-barwidth/2
+        length_orth=3.5
+        if six_width: length_orth=2.880 
+
         scinBox = geom.shapes.Box( 'scinbox'+self.name,
-                                    dx = 0.5*Q("0.036m"),   # width single bar
+                                    dx = 0.5*Q(f"{barwidth}m"),   # width single bar
                                     dy = 0.5*Q("3.300m"),   # length bar for stereo bars
-                                    dz = 0.5*Q("0.017m"))   # thickness single bar
+                                    dz = 0.5*Q("0.016m"))   # thickness single bar
```

Comment:
If the bar thickness is used in multiple places, it should also be a variable that you only set once.